### DISCUSSION TITLE
regarding the name changes

### DISCUSSION THREAD

**Discussion by vishalsaraha** [magicuidesign/magicui#371] [General]
the component path is getting error every time when i'm using it can you guys tell i need to do

**Comment by dillionverma** [comment]
Hi! Which component are you referring to?

**Comment by Yeom-JinHo** [comment]
@vishalsaraha  Could you provide more details about component name and the exact error message?

**Comment by sharaddh** [comment]
@vishalsaraha may be the error is raising on pnpm build and files is missing, you should drop the file address like this-
 {
    name: "time-scale-filter-demo",
    type: "registry:example",
    description: "Time Scale Filter demo",
    registryDependencies: ["@magicui/time-scale-filter"],
    files: [
      {
        path: "example/time-scale-filter-demo.tsx",
        type: "registry:example",
      },
    ],
  },
